vnfs/libexec/wwmkchroot/include-rhel

```diff
@@ -1,30 +1,56 @@
-if [ "$PKG_MGR" = "dnf" ]; then
-    YUM_CONF="/etc/dnf/dnf.conf"
-    YUM_CMD="dnf -y -c $CHROOTDIR/$YUM_CONF --installroot $CHROOTDIR --setopt=reposdir=$CHROOTDIR/etc/yum.repos.d $EXTRA_OPTS" 
-    YUM_REPOCMD="dnf config-manager -y -c $CHROOTDIR/$YUM_CONF --installroot $CHROOTDIR --setopt=reposdir=$CHROOTDIR/etc/yum.repos.d $EXTRA_OPTS"
-else
-    PKG_MGR="yum"    
-    YUM_CONF="/etc/yum.conf"
-    YUM_CMD="yum -y -c $CHROOTDIR/$YUM_CONF --tolerant --installroot $CHROOTDIR $EXTRA_OPTS"
-    YUM_REPOCMD="yum-config-manager -y -c $CHROOTDIR/$YUM_CONF --tolerant --installroot $CHROOTDIR $EXTRA_OPTS"
-fi
+#!/bin/bash
```

Comment:
Same as above, no need to add a #! line back into a script that is only ever sourced.